python/src/energuide/extracted_datatypes.py

```diff
@@ -506,3 +514,51 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
             'areaMetres': self.height * self.width,
             'areaFeet': self.height * self.width * _FEET_SQUARED_MULTIPLIER
         }
+
+
+class Ventilation(_Ventilation):
+    _TYPE_MAP = {
```

Comment:
I appreciate what this is trying to do, but it isn't very clear. Can you convert this into a private function that accepts two arguments as bools?